js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -341,7 +358,34 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
           ranges={ranges}
           setRanges={setRanges}
         />
+
+        <Box flex={{direction: 'column', gap: 32}} style={{marginTop: 32}}>
+          <DialogBodySection title="Tags">
+            <TagEditor
+              tagsFromSession={tags}
+              onChange={setTags}
+              open={tagEditorOpen}
+              onRequestClose={() => setTagEditorOpen(false)}
+            />
+            {tags.length ? (
+              <div style={{border: `1px solid ${Colors.Gray300}`, borderRadius: 8, padding: 3}}>
+                <TagContainer tagsFromSession={tags} onRequestEdit={() => setTagEditorOpen(true)} />
+              </div>
```

Comment:
Can't see what this looks like in the current screenshots, can you add one with the tag container visible?

Reply:
Ahh yep it's this one - not super glamorous but the same as the op-job backfill UI 

<img width="880" alt="image" src="https://user-images.githubusercontent.com/1037212/212175779-0f5b8d37-41a2-4d61-8711-cd2927ceb1da.png">